### DISCUSSION TITLE
Установка на hetzner cloud

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#227] [Как это сделать?] [answered]
Скажите пожалуйста, подходит ли сервер hetzner для установки totum?
и ставил ли кто-то туда систему?

https://www.hetzner.com/cloud

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Их документация говорит, что 20 Ubuntu у них есть:

<img width="798" alt="Снимок экрана 2022-09-28 в 16 06 35" src="https://user-images.githubusercontent.com/55755565/192786390-35f23bed-87f6-43c0-b9e8-465747a4e950.png">

Минимальный тариф на который бы я ставил это CPX11 с 2 vCPU — стоит сейчас ~6 EUR

Если для теста не хотите покупать домен, то поставьте без домена https://ru.docs.totum.online/ubuntu#without-domain, если работает, то перезальете образ и поставите уже чистовую.
